README.md

```diff
@@ -1,24 +1,49 @@
-# README
+# SloFo
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is midcourse project for CraftAcademy Class Feb-april 2017
 
-Things you may want to cover:
 
-* Ruby version
+**Pivotal Tracker**
+[Pivotal Tracker for SloFo](https://www.pivotaltracker.com/n/projects/1996373).
+
+**GitHub**
+This site base reside on [GitHub Pages](https://github.com/CraftAcademy/slow_food_online_feb_17).
+
+### Installing
+
+Fork and clone repo, then run:
+
+```
```

Comment:
just for your information you can always add the type of hightlighting you want for the code for example in this scenario you can add the word `bash` after the triple backticks ``` bash